gpkit/tests/t_constraints.py

```diff
@@ -358,9 +358,23 @@ def test_substitution_issue905(self):
         sol = bm.solve(verbosity=0)
         self.assertAlmostEqual(sol["cost"], 1.0)
 
+class TestLocalsolve(unittest.TestCase):
+    """Test different SP solution methods"""
+
+    def test_penaltyccp(self):
```

Comment:
Add underscore to match method name?

```suggestion
    def test_penalty_ccp(self):
```